apidoc/Titanium/UI/Android/Toolbar.yml

```diff
@@ -0,0 +1,255 @@
+---
+name: Titanium.UI.Android.Toolbar
+summary: An Android toolbar, which can contain buttons and other custom views.
+description: |
+    A `Toolbar` is created by the <Titanium.UI.Android.createToolbar> factory method.
+    
+    A `Toolbar` can contain buttons, as well as certain other widgets, including text fields and labels. 
+    
+    A toolbar can be positioned everywhere in the the layout as a [View](Titanium.UI.View). It can also be used as an ActionBar 
+    for [activities](Titanium.Android.Activity). This allows a toolbar to inherit [ActionBar's](Titanium.Android.ActionBar) methods,
+    properties and events and provide a better customization of this design element. For example you can provide your own images
+    to be used as navigation button icon, nverflow menu icon and logo. In order to do that an application must be using Theme.AppCompat.NoTitleBar.
```

Comment:
Also, the following needs a comma. Please change...
`In order to do that an application...`
...to...
`In order to do that, an application...`